pkg/virt-controller/network/pod_annotations.go

```diff
@@ -31,19 +33,31 @@ import (
 )
 
 func GeneratePodAnnotations(networks []virtv1.Network, interfaces []virtv1.Interface, multusStatusAnnotation string, bindingPlugins map[string]virtv1.InterfaceBindingPlugin) map[string]string {
-	newAnnotations := map[string]string{}
+	var err error
+
+	var ifacePciMap map[string]*networkv1.DeviceInfo
 	if vmispec.SRIOVInterfaceExist(interfaces) {
-		networkPCIMapAnnotationValue := deviceinfo.CreateNetworkPCIAnnotationValue(
```

Comment:
The old annotation should still be updated on existing virt-launcher pods. Otherwise, they will be broken.
SRIOV hotplug and maybe even migration will fail.